code/iaas/service-discovery/server/src/main/java/io/cattle/platform/servicediscovery/service/ServiceDiscoveryServer.java

```diff
@@ -0,0 +1,17 @@
+package io.cattle.platform.servicediscovery.service;
+
+import io.cattle.platform.core.model.Service;
+
+import java.util.AbstractMap.SimpleEntry;
+import java.util.List;
+import java.util.Map;
+
+public interface ServiceDiscoveryServer {
```

Comment:
This is more of a nitpick, but can we rename this to ServiceDiscoveryService.
When I see Server, I think of a daemon process.